NestedConditionalStatements3/04.CarToGo/04.CarToGo.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace _04.CarToGo
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            double budget = double.Parse(Console.ReadLine());
+            string summerOrWinter = Console.ReadLine();
+
+            string Class = string.Empty;
+            string car = string.Empty;
+            double price = 0;
+
+            if (budget > 0 && budget <= 100)
+            {
+                Class = "Economy class";
+
+                if (summerOrWinter == "Summer")
```

Comment:
use const for "Summer" or define bool variable. If it is changed you kneed to change it in 5 places